meta-mender-qemu/recipes-connectivity/openssl/openssl_%.bbappend

```
@@ -0,0 +1,17 @@
+
+do_install_append () {
+	local cnf="${D}${sysconfdir}/ssl/openssl.cnf"
+	local level_set=""
+
+	if [ -d "${D}${sysconfdir}/ssl" ]; then
+		[ -f "${cnf}" ] && grep -q '^[ ]*CipherString[ ]*=[ ]*DEFAULT@SECLEVEL=2' "${cnf}" && level_set="2"
+		if [ "${level_set}" == "" ]; then
+				# see MEN-3730, this is to align with the current default settings on raspbian
+				cat >> "${cnf}" <<EOF
+[system_default_sect]
+MinProtocol = TLSv1.2
+CipherString = DEFAULT@SECLEVEL=2
+EOF
+		fi
+	fi
+}
```

We know exactly the contents of `openssl.cnf` from upstream, so we don't need to check if the file exists nor if SECLEVEL is already set. And actually, we want to make sure that this `do_install_append` breaks if the file ever gets moved, for example (as it is now I would silently pass and do nothing). That is the best practice in yocto.

Therefore I suggest a simple:

```suggestion
do_install_append () {
	# see MEN-3730, this is to align with the default settings on Raspbian
	cat >> "${D}${sysconfdir}/ssl/openssl.cnf" <<EOF
[system_default_sect]
MinProtocol = TLSv1.2
CipherString = DEFAULT@SECLEVEL=2
EOF
}
```